images/prometheus-config-reloader/config/main.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+variable "extra_packages" {
+  description = "The additional packages to install"
+  default     = ["prometheus-config-reloader", "prometheus-config-reloader-oci-entrypoint-compat"]
```

Comment:
If this is something we expect to always need in the image, regardless of what `prometheus-config-reloader` package variant we install, we can just include that in the latest.apko.yaml `packages` section, so callers only have to pass `extra_packages = ["prom-config-reloader-variant-foo"]`

Reply:
So you are suggesting me to do?:
```
contents:
  packages:
    - prometheus-config-reloader-oci-entrypoint-compat
  ```

Reply:
Just attempted to fix it, feel free to review @imjasonh , thanks so much!